packages/@sanity/code-input/src/CodeInput.js

```diff
@@ -64,6 +67,31 @@ export default class Code extends React.PureComponent {
     ]))
   }
 
+  handleToggleSelectLine = line => {
+    const {type, onChange} = this.props
+    const path = ['highlightedLines']
+
+    const highlightedLines = xor(this.props.value.highlightedLines, [line]).sort(compareNumbers)
+
+    onChange(PatchEvent.from([
+      setIfMissing({_type: type.name, highlightedLines: []}),
+      line ? set(highlightedLines, path) : unset(path)
+    ]))
+
+  }
+
+  handleEditorLoad = editor => {
+    editor.focus()
+    editor.on('guttermousedown', event => {
+      const target = event.domEvent.target
+      if (target.className.indexOf('ace_gutter-cell') == -1) {
```

Comment:
`classList` is supported from IE11 and above, so better to use `target.classList.contains('ace_gutter-cell')` here.